apps/frontend/components/field/referee/prestart.tsx

```diff
@@ -40,6 +44,11 @@ const MatchPrestart: React.FC<MatchPrestartProps> = ({
               onChange={present => {
                 updateMatchParticipant({ present });
                 if (updateInspectionStatus) updateInspectionStatus(null);
+                if (present === 'present') {
+                  setInspectionStartTime(new Date().toISOString());
```

Comment:
this statement can be reduced to `setInspectionStartTime(present === 'present' ? Date.now() : null)`